tests/es/test_client.py

```diff
@@ -150,3 +154,44 @@ async def does_not_raise():
     await does_not_raise()
 
     assert mock_func.call_count == 1
+
+
+@pytest.mark.parametrize(
+    "enabled_license, licenses_enabled",
+    [
+        (License.BASIC, [License.BASIC]),
+        (License.GOLD, [License.BASIC, License.GOLD]),
+        (License.PLATINUM, [License.BASIC, License.GOLD, License.PLATINUM]),
+        (License.ENTERPRISE, [License.BASIC, License.GOLD, License.PLATINUM, License.ENTERPRISE]),
+        (License.TRIAL, [License.BASIC, License.GOLD, License.PLATINUM, License.ENTERPRISE, License.TRIAL])
+    ]
+)
+@pytest.mark.asyncio
+async def test_has_license_enabled(enabled_license, licenses_enabled):
```

Comment:
I feel it may be more cleaner to have another test function `test_has_license_disabled`

Reply:
I can split it up 👍